infra/smoke-test/test/cache-unzip.ts

```diff
@@ -16,9 +16,12 @@ t.test(
         // time but should be enough for a small install.
         await setTimeout(1000)
 
-        const cacheEntries = readdirSync(join(dirs.cache, 'vlt'), {
-          withFileTypes: true,
-        })
+        const cacheEntries = readdirSync(
+          join(dirs.cache, 'vlt/registry-client'),
```

Comment:
Does this PR change the location of our cache directory?

Reply:
Previously, the config would override the xdg location, and would default to having 'registry-client' in the path. But if configured, it would not. Now, the default is just xdg.cache(), and registry-client is tacked on regardless of whether it's the default or the configured value.